applications/Deitos_Network.md

```diff
@@ -273,15 +279,20 @@ Elements related to data consumption or querying, as well as the inclusion of ot
 
 ## Mid-Term Plans
 
+### High Overview roadmap.
```

Comment:
```suggestion
### High-Level Overview roadmap.
```

Reply:
Not sure if you meant high-level here, feel free to accept or ignore the changes.